### ISSUE TITLE
sf org display user returns duplicate "Profile Name"

### ISSUE THREAD

**Issue by ghost** [forcedotcom/cli#2762]
> **Note** 
> Before you submit your issue, make sure that:
> - You're using the latest version of Salesforce CLI.
> - You've searched both open and closed issues for related posts.
> - You've used the `doctor` command to diagnose common issues. 
> - You understand that GitHub Issues don't adhere to any agreement or SLA.
>   - If you require immediate assistance, use official channels such as Salesforce Customer Support.

### Summary
Running the sf command `sf org display user -o <target-org>` outputs duplicate "Profile Name" info.

### Steps To Reproduce
1. Authenticate with an org.
2. Run the following sf command `sf org display user -o <target-org>`

### Expected result
User details are displayed in stdout.

### Actual result
User details are displayed in stdout, however with duplicate "Profile Name" lines.

### System Information
```json
{
  "architecture": "darwin-arm64",
  "cliVersion": "@salesforce/cli/2.30.8",
  "nodeVersion": "node-v21.6.2",
  "osVersion": "Darwin 23.1.0",
  "rootPath": "/Users/uber/.nvm/versions/node/v21.6.2/lib/node_modules/@salesforce/cli",
  "shell": "zsh",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 3.0.9 (core)",
    "@oclif/plugin-commands 3.1.4 (core)",
    "@oclif/plugin-help 6.0.13 (core)",
    "@oclif/plugin-not-found 3.0.11 (core)",
    "@oclif/plugin-plugins 4.2.5 (core)",
    "@oclif/plugin-search 1.0.16 (core)",
    "@oclif/plugin-update 4.1.13 (core)",
    "@oclif/plugin-version 2.0.12 (core)",
    "@oclif/plugin-warn-if-update-available 3.0.12 (core)",
    "@oclif/plugin-which 3.1.0 (core)",
    "@salesforce/cli 2.30.8 (core)",
    "apex 3.0.25 (core)",
    "auth 3.3.13 (core)",
    "data 3.1.1 (core)",
    "deploy-retrieve 3.2.16 (core)",
    "info 3.0.27 (core)",
    "limits 3.1.10 (core)",
    "marketplace 1.0.25 (core)",
    "org 3.3.14 (core)",
    "packaging 2.1.10 (core)",
    "schema 3.1.3 (core)",
    "settings 2.0.27 (core)",
    "sobject 1.1.13 (core)",
    "source 3.1.15 (core)",
    "telemetry 3.1.13 (core)",
    "templates 56.0.16 (core)",
    "trust 3.3.10 (core)",
    "user 3.2.11 (core)"
  ]
}
```
### Additional information
<img width="1414" alt="Screenshot 2024-03-10 at 21 00 48" src="https://github.com/forcedotcom/cli/assets/37094458/94d22f05-066f-4641-ba89-cc0da44aa875">

**Comment by shetzel** [comment]
@dwbuttler - thanks for reporting and fixing the bug!  It'll be in the next CLI release candidate in a few days, as well as a shoutout to you in the CLI release notes.

**Comment by jshackell-sfdc** [comment]
This issue is fixed in Salesforce CLI release 2.33.3.
